packages/eslint-plugin-twenty/src/tests/no-spread-props.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { RuleTester } from "@typescript-eslint/rule-tester";
+
+import noSpreadPropsRule from "../rules/no-spread-props";
+
+const ruleTester = new RuleTester({
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: "./tsconfig.json",
+    tsconfigRootDir: __dirname,
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+});
+
+ruleTester.run("no-spread-props", noSpreadPropsRule, {
+  valid: [
```

Comment:
Clean PR, thank you,

Could you add a test with : `{...{ prop1, prop2 }}` ?

Reply:
@lucasbordeau The test has been added